rust/operator-binary/src/authentication/password/ldap.rs

```diff
@@ -30,16 +22,26 @@ const LDAP_PASSWORD_ENV: &str = "LDAP_PASSWORD";
 pub enum Error {
     #[snafu(display("Trino does not support unverified TLS connections to LDAP"))]
     UnverifiedLdapTlsConnectionNotSupported,
+
+    #[snafu(display("Failed to construct LDAP endpoint URL"))]
+    LdapEndpoint {
+        source: stackable_operator::commons::authentication::ldap::Error,
+    },
+
+    #[snafu(display("Failed to construct LDAP volumes and volume mounts"))]
+    LdapVolumesAndMounts {
+        source: stackable_operator::commons::authentication::ldap::Error,
+    },
```

Comment:
In the Kafka PR it was Volume and VolumeMounts (granted, another error). Should we maybe add something to the code guidelines to be consistent here?

Edit: its Volume and VolumeMounts in authentication/password/mod.rs as well.

Reply:
https://github.com/stackabletech/trino-operator/pull/522/commits/2a9368701cc44c9b268ea0f7dd7b464576995165